test/unit/test_label_procs.py

```diff
@@ -208,9 +208,20 @@ def test_smoke_dynamic_grouped_label(conn, timestamp_string):
 # Test that validates that correct error message was returned
 @pytest.mark.parametrize("statement, expected_error", test_cases)
 def test_error_message(conn, timestamp_string, statement, expected_error):
+    # Create a label with this name
+    old_label = generate_unique_name("old_label", timestamp_string)
+    sql = f"call ADMIN.CREATE_LABEL('{old_label}', NULL, NULL, 'rows_produced > 100');"
+    assert run_proc(conn, sql) is None, "ADMIN.CREATE_LABEL did not return NULL value!"
+
+    # Create a dynamic label
+    old_dyn_label = generate_unique_name("old_dyn_label", timestamp_string)
+    sql = f"call ADMIN.CREATE_LABEL(NULL, '{old_dyn_label}', NULL, 'QUERY_TYPE', TRUE);"
+    assert run_proc(conn, sql) is None, "ADMIN.CREATE_LABEL did not return NULL value!"
 
     label = generate_unique_name("label", timestamp_string)
-    sql = statement.format(label=label)
+    sql = statement.format(
+        label=label, old_label=old_label, old_dyn_label=old_dyn_label
+    )
```

Comment:
The python label implementation validated the conditions first and then verified existing labels against the input (e.g. cannot create a label with the same name as one that exists, a label has to exist with the name being updated).

I can separate this change out to its own PR, but hopefully this is clear enough.